metadata.rb

```diff
@@ -5,7 +5,7 @@
 description      'Installs/Configures mailhog'
 version          '0.1.0'
 
-depends          'runit', '>= 0.0.0'
+depends          'runit', '= 1.7.4'
```

Comment:
Hmmm. It doesn't play nice with some of my vagrant packages.
Is there any reason to lock it on 1.7.4?